lib/libyui_client/validate.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Validate
+    module_function
+
+    def filter(filter)
+      filters = %i[id label type].freeze
```

Comment:
We should freeze these constants on the class level and not each time we create object.